src/ComScoreStreamingAnalytics.ts

```diff
@@ -137,6 +180,14 @@ export class ComScoreStreamingAnalytics {
     this.transitionToVideo();
   }
 
+  private stopComScoreTrackingBackgroundAd(): void {
```

Comment:
Why not using the existing `stopComScoreTracking` method? Seems to do the same but also stets the state.